lib/node_modules/@stdlib/blas/ext/base/dapxsumkbn/README.md

```diff
@@ -144,8 +146,123 @@ console.log( v );
 
 <!-- /.examples -->
 
+<!-- C interface documentation. -->
+
 * * *
 
+<section class="c">
+
+## C APIs
+
+<!-- Section to include introductory text. Make sure to keep an empty line after the intro `section` element and another before the `/section` close. -->
+
+<section class="intro">
+
+</section>
+
+<!-- /.intro -->
+
+<!-- C usage documentation. -->
+
+<section class="usage">
+
+### Usage
+
+```c
+#include "stdlib/blas/ext/base/dapxsumkbn.h"
+```
+
+#### c_dapxsumkbn( N, alpha, \*X, strideX )
+
+Adds a scalar constant to each double-precision floating-point strided array element and computes the sum using an improved Kahan–Babuška algorithm.
+
+```c
+const double x[] = { 1.0, 2.0, 3.0, 4.0 };
+
+double v = c_dapxsumkbn( 4, 5.0, x, 1 );
+```
+
+The function accepts the following arguments:
+
+-   **N**: `[in] CBLAS_INT` number of indexed elements.
+-   **alpha**: `[in] double` scalar constant.
+-   **X**: `[in] double*` input array.
+-   **strideX**: `[in] CBLAS_INT` index increment for `X`.
+
+```c
+double c_dapxsumkbn( const CBLAS_INT N, const double alpha, const double *X, const CBLAS_INT strideX );
+```
+
+#### c_dapxsumkbn_ndarray( N, alpha, \*X, strideX, offsetX )
+
+Adds a scalar constant to each double-precision floating-point strided array element and computes the sum using an improved Kahan–Babuška algorithm and alternative indexing semantics.
+
+```c
+double x[] = { 1.0, 2.0, 3.0, 4.0 };
+
+double v = c_dapxsumkbn_ndarray( 4, 5.0, x, 1, 0 );
+```
+
+The function accepts the following arguments:
+
+-   **N**: `[in] CBLAS_INT` number of indexed elements.
+-   **alpha**: `[in] double` scalar constant.
+-   **X**: `[inout] double*` input array.
```

Comment:
```suggestion
-   **X**: `[in] double*` input array.
```

In all C function signatures throughout this PR, `X` should be annotated as `const`.